src/DisplayTotalMasteryPage/DisplayTotalMasteryPage.tsx

```diff
@@ -0,0 +1,92 @@
+import { Container, makeStyles, Theme, Typography } from '@material-ui/core'
+import React, { FC, useContext, useState } from 'react'
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+import { GlobalContext } from '../GlobalContext'
+import { theme } from '../theme'
+
+interface DisplayTotalMasteryPageProps {}
+
+const useStyles = makeStyles({
+	resultRow: (theme: Theme) => ({
+		alignSelf: 'stretch',
+		border: '1px solid',
+		borderCollapse: 'collapse',
+		borderColor: theme.palette.primary.main,
+		display: 'flex',
+		justifyContent: 'space-between',
+		marginBottom: theme.spacing(3),
+	}),
+})
+
+const DisplayTotalMasteryPage: FC<DisplayTotalMasteryPageProps> = ({}) => {
```

Comment:
Let's shorten this component name a bit

```suggestion
const MasteryPage: FC<MasteryPageProps> = () => {
```

Reply:
Make sure the file and its folder get renamed too! (`MasteryPage`)